schema/schema.go

```diff
@@ -196,7 +196,11 @@ func (s *Schema) Decode(row []string, out interface{}) error {
 		fieldValue := outv.Field(i)
 		if fieldValue.CanSet() { // Only consider exported fields.
 			field := outt.Field(i)
-			f, fieldIndex := s.GetField(field.Name)
+			fieldName, ok := field.Tag.Lookup("tableheader")
```

Comment:
Could you please make tableheader a constant?

Reply:
Also thought about this.